modules/dnn/src/int8layers/softmax_layer.cpp

```diff
@@ -22,15 +22,29 @@ class SoftMaxLayerInt8Impl CV_FINAL : public SoftmaxLayerInt8
 
     SoftMaxLayerInt8Impl(const LayerParams& params)
     {
-        axisRaw = params.get<int>("axis", 1);
+        setParamsFrom(params);
+
+        axis = params.get<int>("axis", 1);
         logSoftMax = params.get<bool>("log_softmax", false);
+        coerced_2d = params.get<bool>("coerced_2d", false);
 
         input_sc = params.get<float>("input_scale");
         input_zp = params.get<int>("input_zeropoint");
 
         output_sc = params.get<float>("scales");
         output_zp = params.get<int>("zeropoints");
-        setParamsFrom(params);
+
+        if (blobs.empty()) // if no lookUpTable is found
+        {
+            Mat lookUpTable(1, 256, CV_32F);
```

Comment:
Since the The softmax layer has a high correlation with model accuracy, I propose to change the `lookUpTable` from `float` to `double`.

Reference from ORT: https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cpu/quantization/qlinear_softmax.cc#L37

Reply:
Well, if you see [line 45 from your reference](https://github.com/microsoft/onnxruntime/blob/76fd9aa745db596d5a1a8053fa76a1f0c92cd1b7/onnxruntime/contrib_ops/cpu/quantization/qlinear_softmax.cc#L45), it casts back to float in the end. I do not think the loss of accuracy comes from the float lookup table, but the cast from float -> int32_t -> int8_t if we are [talking about onnxruntime](https://github.com/microsoft/onnxruntime/blob/76fd9aa745db596d5a1a8053fa76a1f0c92cd1b7/onnxruntime/contrib_ops/cpu/quantization/qlinear_softmax.cc#L243-L247).